docs/install.md

```diff
@@ -1,125 +1,103 @@
+    kind: Subscription
+    metadata:
+      name: aws-load-balancer-operator
+      namespace: aws-load-balancer-operator
+    spec:
+      channel: stable-v1
+      name: aws-load-balancer-operator
+      source: redhat-operators
+      sourceNamespace: openshift-marketplace
+      config:
+        env:
+        - name: ROLEARN
+          value: "${ROLEARN}"
+    EOF
     ```
 
-    For each `CredentialsRequest` object, `ccoctl` creates an IAM role with a trust
-    policy that is tied to the specified OIDC identity provider, and permissions
-    policy as defined in each `CredentialsRequest` object. This also generates a set
-    of secrets in a **manifests** directory that is required
-    by the **aws-load-balancer-controller**.
-
-4. Apply the secrets to your cluster:
-
-    ```bash
-    ls manifests/*-credentials.yaml | xargs -I{} oc apply -f {}
-    ```
+The **aws-load-balancer-operator** will wait until the required secret is created before moving to the available state.
 
-5. Verify that the corresponding **aws-load-balancer-controller** pod was created:
+### Post operator installation on STS cluster
+In an STS cluster, the `CredentialsRequest` needs to be set with the IAM role which needs to be provisioned manually.
 
-    ```bash
-    oc -n aws-load-balancer-operator get pods
-    NAME                                                            READY   STATUS    RESTARTS   AGE
-    aws-load-balancer-controller-cluster-9b766d6-gg82c              1/1     Running   0          137m
-    aws-load-balancer-operator-controller-manager-b55ff68cc-85jzg   2/2     Running   0          3h26m
-    ```
+There are two options for creating the controller's IAM role:
+1. Using [`ccoctl`](https://docs.openshift.com/container-platform/latest/authentication/managing_cloud_provider_credentials/cco-mode-sts.html#cco-ccoctl-configuring_cco-mode-sts) and a pre-defined `CredentialsRequest`.
+2. Using AWS CLI and pre-defined AWS manifests.
 
-#### Use a Predefined Secret
-In some cases, the provisioning of the credentials secret cannot be done by the **cloud-credential-operator**, e.g. ROSA user clusters are forbidden to create `CredentialsRequest`.
-Instead, the secret can be explicitly referenced in the `AWSLoadBalancerController` CR, see [credentials.name field description](./tutorial.md#credentialsname).
-The credentials secret needs to reference a role with all the policies needed by the controller.
-There are two options for creating the credentials secret.  Use Option 1 with `ccoctl`, or Option 2 with the AWS CLI.
+If your system doesn't support `ccoctl`, the second option is the only available choice.
 
-##### Option 1. Using `ccoctl`
-The dedicated controller's `CredentialsRequest` is maintained in [hack/controller](../hack/controller/) directory of this repository.
+#### Option 1. Using `ccoctl`
+The controller's `CredentialsRequest` is maintained in [hack/controller](../hack/controller/) directory of this repository.
```

Comment:
Is there a reason you linked to the directory and not just the file itself?

Reply:
No particular reason. Let me update to a file.

Reply:
Done.